pkg/services/navtree/navtreeimpl/applinks.go

```diff
@@ -163,6 +163,20 @@ func (s *ServiceImpl) processAppPlugin(plugin pluginstore.Plugin, c *contextmode
 
 	s.addPluginToSection(c, treeRoot, plugin, appLink)
 
+	if plugin.ID == "grafana-slo-app" {
+		// Add Service Center as a standalone nav item under Alerts & IRM
+		if alertsSection := treeRoot.FindById(navtree.NavIDAlertsAndIncidents); alertsSection != nil {
+			serviceLink := &navtree.NavLink{
+				Text:       "Service Center",
+				Id:         "standalone-plugin-page-slo-services",
+				Url:        s.cfg.AppSubURL + "/a/grafana-slo-app/services",
+				SortWeight: 1,
+				IsNew:      true,
+			}
+			alertsSection.Children = append(alertsSection.Children, serviceLink)
+		}
+	}
```

Comment:
i _think_ you can do this using the `navigationAppPathConfig` on line 340? probably something like:
```go
"/a/grafana-slo-app/services": {SectionID: navtree.NavIDAlertsAndIncidents, SortWeight: 1},
```

only thing i'm not sure about is whether that supports the `isNew` property

Reply:
Yea, exactly as @ashharrison90 says, you could do this via configuration as well. The `isNew` property is not supported yet, but we _could_ add support for it. 

You could also do this via configuration ([sample.ini](https://github.com/grafana/grafana/blob/180f579f18bfb4fbd51495497019a489587f1e1f/conf/sample.ini#L1949)), but again, the `isNew` wouldn't work yet. Example.:

```ini
[navigation.app_standalone_pages]
/a/grafana-slo-app/services = alerts-and-incidents 2 ; 2 is the sort weight
```

I have some concerns about adding this "specificity" to this otherwise generic function (although we have other plugin-specific parts down in the file).

Reply:
I tried the `navigationAppPathConfig` but as you say it doesn't support the `isNew`. 

Furthermore, this is a temporary ™️  solution as we will move the service center out of the slo app in the next quarter.